mlrun/api/api/endpoints/submit.py

```diff
@@ -75,8 +75,7 @@ async def submit_job(
             auth_info,
         )
     if data.get("schedule"):
-        await fastapi.concurrency.run_in_threadpool(
-            mlrun.api.utils.auth.verifier.AuthVerifier().query_project_resource_permissions,
+        await mlrun.api.utils.auth.verifier.AuthVerifier().query_project_resource_permissions(
```

Comment:
below you missed to `run_in_threadpool` the `chief_client.submit_job` or more correctly I missed it when adding it to that async endpoint

Reply:
gotit :)